assets/queries/cloudFormation/vpc_has_FlowLog_disabled/query.rego

```diff
@@ -0,0 +1,34 @@
+package Cx
+
+CxPolicy [ result ] {
+  resource := input.document[i].Resources[name]
+  resource.Type = "AWS::EC2::FlowLog"
```

Comment:
I think you should try the other way around. Find the VPC first, then check if there's any resources of type `AWS::EC2::FlowLog` in the same document

Reply:
Done